lib/contracts/contribution.js

```diff
@@ -1,20 +1,20 @@
 const ethers = require('ethers');
-const RSVP = require('rsvp');
 
 const ContributionSerializer = require('../serializers/contribution');
 const Base = require('./base');
 
 class Contribution extends Base {
   all() {
     return this.functions.contributionsCount()
-      .then(count => {
+      .then(async (count) => {
         let contributions = [];
 
         for (let id = 1; id <= count; id++) {
-          contributions.push(this.getById(id));
+          const contribution = await this.getById(id)
```

Comment:
❤️

Reply:
to promise or not to promise? :bumiwonders:

Reply:
If you can wait for it you don't need to promise anything.